src/app/accounts/register.component.ts

```diff
@@ -19,6 +19,7 @@ import { MasterPasswordPolicyOptions } from 'jslib/models/domain/masterPasswordP
 import { Policy } from 'jslib/models/domain/policy';
 
 import { PolicyData } from 'jslib/models/data/policyData';
+import { ReferenceEventData } from 'jslib/models/domain/referenceEventData';
```

Comment:
per other PRs, path will change to API request model

Reply:
Fixed